packages/api/src/controllers/ValidatorsController.js

```diff
@@ -24,13 +35,35 @@ class ValidatorsController {
 
     const isActive = validators.some(validator => validator.pro_tx_hash === hash)
 
+    const [host] = proTxInfo?.state.service ? proTxInfo?.state.service.match(/^\d+\.\d+\.\d+\.\d+/) : [null]
+    const [servicePort] = proTxInfo?.state.service ? proTxInfo?.state.service.match(/\d+$/) : [null]
+
+    const coreStatus = await checkTcpConnect(servicePort, host)
+    const platformStatus = await checkTcpConnect(proTxInfo?.state.platformP2PPort, host)
+    const grpcStatus = await checkTcpConnect(proTxInfo?.state.platformHTTPPort, host)
+
+    const endpoints = {
+      coreP2P: new ConnectionData(`${host}:${servicePort}`, coreStatus),
```

Comment:
No need for an separate model here, since its not really an entity and not used in other endpoints

just make a structure {
host: "",
port: "",
status: ""
}